src/sknnr/transformers/_ccora.py

```diff
@@ -4,8 +4,8 @@
 
 def ftest_cor(p, q, N, cor):
     """Direct port of yaImpute ftest_cor in yai
-    REFERENCE: https://github.com/jeffreyevans/yaImpute/blob/f06d1daf13cf461730c3da5f18f022c1074e9826/R/yai.R#L11-L42 # noqa: E501
-    """
+    REFERENCE: https://github.com/jeffreyevans/yaImpute/blob/f06d1daf13cf461730c3da5f18f022c1074e9826/R/yai.R#L11-L42
+    """  # noqa: E501
```

Comment:
Interesting, but probably not a big deal, right?

Reply:
Yeah, that's what I thought. I prefer the other a little, but it's a pretty minor drawback.